data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/cloudevents/PartitionKey.java

```diff
@@ -0,0 +1,27 @@
+package dev.knative.eventing.kafka.broker.core.cloudevents;
+
+import io.cloudevents.CloudEvent;
+
+public final class PartitionKey {
+
+  public static final String PARTITION_KEY_KEY = "partitionkey";
+
+  public static final String PARTITION_KEY_DEFAULT = "";
+
+  /**
+   * Extract the partitionkey extension from the given event.
+   *
+   * @param event event from which extracting partition key.
+   * @return partitionkey extension value or PARTITION_KEY_DEFAULT if not set
+   * @link https://github.com/cloudevents/spec/blob/master/extensions/partitioning.md
+   */
+  public static String extract(final CloudEvent event) {
```

Comment:
Why don't you just return null?

Reply:
Done: https://github.com/knative-sandbox/eventing-kafka-broker/pull/50/commits/fb22a92ee9e995cf973a83be383f1b8678861e0b